File: phase01/immortals_repo/harness/scenarioconductor/data/base/testharnessconfiguration.py
class TestHarnessConfiguration:
    """
    :type enabled: bool
    :type protocol: str
    :type url: str
    :type port: int
    """

    @classmethod
    def from_dict(cls, d):
        return cls(
            enabled=d['enabled'],
            protocol=d['protocol'],
            url=d['url'],
            port=d['port']
        )

    def __init__(self,
                 enabled,
                 protocol,
                 url,
                 port
                 ):
        self.enabled = enabled
        self.protocol = protocol
        self.url = url
        self.port = port

File: immortals_repo/buildSrc/ImmortalsConfig/src/main/java/mil/darpa/immortals/config/TestHarnessConfiguration.java
package mil.darpa.immortals.config;

/**
 * Created by awellman@bbn.com on 11/1/17.
 */
public class TestHarnessConfiguration {
    private String protocol = "http";
    private String url = "brass-th";
    private int port = 80;

    TestHarnessConfiguration() {
    }

    public String getProtocol() {
        return protocol;
    }

    public String getUrl() {
        return url;
    }

    public int getPort() {
        return port;
    }
}
